src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISFunctions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using Microsoft.AspNetCore.HttpSys.Internal;
+using static Microsoft.AspNetCore.Server.IISIntegration.IISDelegates;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration
+{
+    internal unsafe interface IISFunctions
+    {
+        void IISPostCompletion(IntPtr pHttpContext, int cbBytes);
```

Comment:
Since these are a bunch of IIS functions you can remove IIS from each method name.